Restructure (#542)
hacs/documentation: source/docs/use/manage/type/appdaemon.md

AppDaemon apps are apps for [AppDaemon](https://appdaemon.readthedocs.io/en/stable/).

!!! abscract "Not enabled by default"

>     By default, AppDaemon apps are **not** visible in HACS because they are used only by a small subset of users. To see them, you need to enable them in the configuration options.

Author: ```suggestion
    By default, AppDaemon apps are **not** visible in HACS because they are used only by a small subset of users. To see them, you need to enable discovery in the configuration options.

```

> ## Enabling AppDaemon apps in HACS

Author: ```suggestion
## Making AppDaemon apps visible in HACS

```

> To enable AppDaemon apps in HACS, follow these steps:

Author: ```suggestion
To make AppDaemon apps visible in HACS, follow these steps:

```

    By default, AppDaemon apps are **not** visible in HACS because they are used only by a small subset of users. To see them, you need to enable discovery in the configuration options.

## Making AppDaemon apps visible in HACS

To make AppDaemon apps visible in HACS, follow these steps:

> 

Author: ```suggestion

```

    By default, AppDaemon apps are **not** visible in HACS because they are used only by a small subset of users. To see them, you need to enable discovery in the configuration options.

> 

Author: ```suggestion
```

    By default, AppDaemon apps are **not** visible in HACS because they are used only by a small subset of users. To see them, you need to enable discovery in the configuration options.

## Making AppDaemon apps visible in HACS

> 

Author: ```suggestion
```

3. To save the changes, select **Submit**.

> ## AppDaemon apps download location

Author: ```suggestion
## AppDaemon apps download location

Any AppDaemon app downloaded with HACS is stored in your [Home Assistant configuration directory](https://www.home-assistant.io/docs/configuration/#to-find-the-configuration-directory), under  `appdaemon/apps/`.
```

## AppDaemon apps download location

Any AppDaemon app downloaded with HACS is stored in your [Home Assistant configuration directory](https://www.home-assistant.io/docs/configuration/#to-find-the-configuration-directory), under  `appdaemon/apps/`.

> 
> ??? nerd-mode
> 
>     Nothing yet :(

Author: ```suggestion
```

## AppDaemon apps download location

> Any AppDaemon app downloaded with HACS is stored in your [Home Assistant configuration directory](https://www.home-assistant.io/docs/configuration/#to-find-the-configuration-directory), under  `appdaemon/apps/`.

Author: ```suggestion
```
